### ISSUE TITLE
Durable AMQP result backend cannot be disabled

### ISSUE THREAD

**Issue by theappleman** [mher/node-celery#41]
TASK_RESULT_DURABLE is always set to true, even when otherwise specified in Client(options).

From the readme:
`When using AMQP as result backend with celery prior to version 3.1.7 the result queue needs to be non durable or it will fail with a: Queue.declare: (406) PRECONDITION_FAILED.`
```js
var celery = require('node-celery'),
    client = celery.createClient({
        CELERY_TASK_RESULT_DURABLE: false
    });
```

However, the code that has the intention of checking whether the setting has been **set** is testing the boolean value of the option and reversing it:

https://github.com/mher/node-celery/blob/master/celery.js#L30
```js
self.TASK_RESULT_DURABLE = self.TASK_RESULT_DURABLE || true; // Set Durable true by default (Celery 3.1.7)
```
Workaround is to set the fallback to false.

**Comment by mher** [comment]
Can you create a pull request?

**Comment by mher** [comment]
Fixed by #47
